### ISSUE TITLE
Update photometry data model & handler to always use mjd & utc

### ISSUE THREAD

**Issue by acrellin** [skyportal/skyportal#402]
From @profjsb: 

> I think we can safely remove time_format and time_scale. Instead, at the database level, we can assume that mjd is utc and the time format is iso. For now, the API call to post data can say in the schema that those are the formats we're assuming. In the future, we can add optional parameters time_format and time_scale (with defaults to iso and mjd) and we can do the math needed to convert to iso and mjd before updating the table.

Currently, the handler converts all times to iso & tcb

**Comment by acrellin** [comment]
@profjsb @dannygoldstein can you confirm that iso/mjd is the preferred default and not the current iso/tcb?

**Comment by dannygoldstein** [comment]
Confirm

________________________________

**Comment by acrellin** [comment]
@dannygoldstein @profjsb in the context of our usage of astropy here, `time_format` corresponds to one of 
```
            [('jd', astropy.time.formats.TimeJD),
             ('mjd', astropy.time.formats.TimeMJD),
             ('decimalyear', astropy.time.formats.TimeDecimalYear),
             ('unix', astropy.time.formats.TimeUnix),
             ('cxcsec', astropy.time.formats.TimeCxcSec),
             ('gps', astropy.time.formats.TimeGPS),
             ('plot_date', astropy.time.formats.TimePlotDate),
             ('datetime', astropy.time.formats.TimeDatetime),
             ('iso', astropy.time.formats.TimeISO),
             ('isot', astropy.time.formats.TimeISOT),
             ('yday', astropy.time.formats.TimeYearDayTime),
             ('fits', astropy.time.formats.TimeFITS),
             ('byear', astropy.time.formats.TimeBesselianEpoch),
             ('jyear', astropy.time.formats.TimeJulianEpoch),
             ('byear_str', astropy.time.formats.TimeBesselianEpochString),
             ('jyear_str', astropy.time.formats.TimeJulianEpochString)]
```
and `time_scale` corresponds to one of   
```
('tai', 'tcb', 'tcg', 'tdb', 'tt', 'ut1', 'utc', 'local')
```
so it appears that iso and mjd are competing options, but a combination of the two is not available. Or am I just missing something? Given the above, what is the desired format/scale combination? Currently SkyPortal stores all times as iso/tcb.

**Comment by acrellin** [comment]
We'll use mjd/utc

**Comment by dannygoldstein** [comment]
Closed by #191
